neuraxle/pipeline.py

```diff
@@ -267,8 +267,8 @@ def _load_checkpoint(
         self._assign_loaded_pipeline_into_self(loaded_pipeline)
 
         step = self[new_starting_step_index]
-        if isinstance(step, Checkpoint):
-            starting_step_data_container = step.read_checkpoint(starting_step_data_container, context)
+        if isinstance(step, Checkpoint) or (isinstance(step, MetaStepMixin) and isinstance(step.wrapped, Checkpoint)):
```

Comment:
This MUST be fixed or resolved to issue.

Reply:
https://github.com/Neuraxio/Neuraxle/issues/288